packages/serve/startDevServer.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
```

Comment:
Why do you use `js`?

Reply:
@hiroppy Just easier for me to draft it faster. Can change it later once plan is more solidified.